test/it/uk/gov/hmrc/component/CaseSpec.scala

```diff
@@ -482,4 +433,78 @@ class CaseSpec extends BaseFeatureSpec {
 
   }
 
+
+  feature("Get Cases by commodity code") {
+    // TODO
```

Comment:
Presume these will be done as part of this PR?

Reply:
yes. I did not have time yesterday